### PR TITLE
Warning alarm for scan-files timeout

### PR DESCRIPTION
**Pull request by andrewleith**
# Summary | Résumé

Adds a cloudwatch warning alarm for when scan files times out.

### CODE REVIEW

File: aws/eks/cloudwatch_log.tf [hunk 1]
```diff
@@ -45,3 +45,15 @@ resource "aws_cloudwatch_log_metric_filter" "malware-detected" {
     value     = "1"
   }
 }
+
+resource "aws_cloudwatch_log_metric_filter" "scanfiles-timeout" {
+  name           = "scanfiles-timeout"
+  pattern        = "Malware scan timed out for notification.id"
```
Comment (sastels) [inline aws/eks/cloudwatch_log.tf:51]: where does this pattern come from? I only see "Scan timed out for document" in document-download-api.

Comment (andrewleith) [reply aws/eks/cloudwatch_log.tf:51]: This alarm is in anticipation of https://github.com/cds-snc/notification-api/pull/1759.  

My bad, I'll move this to draft until the other PR is merged!

Comment (sastels) [reply aws/eks/cloudwatch_log.tf:51]: 👍 lol no worries!

Comment (jimleroyer) [reply aws/eks/cloudwatch_log.tf:51]: @andrewleith sees in the future. 🔮

Comment (jimleroyer) [reply aws/eks/cloudwatch_log.tf:51]: This could be merged no problem though, right? (but we can wait)

**Review by jimleroyer — APPROVED** [review]
LGTM

File: aws/eks/cloudwatch_log.tf (hunk 1 above)
Comment (andrewleith) [reply aws/eks/cloudwatch_log.tf:51]: Yeah it could, it just wouldnt pick up anything yet.

**Pull request merged**